airflow/providers/amazon/CHANGELOG.rst

```diff
@@ -24,6 +24,45 @@
 Changelog
 ---------
 
+5.1.0
+.....
+
+Breaking changes
```

Comment:
```suggestion
Breaking changes
```

This should be removed (I planned to improve that a bit to not generate that part if needed :) )